### ISSUE TITLE
`authors' was removed in Emacs version 25.1.

### ISSUE THREAD

**Issue by chenyanming** [purcell/package-lint#191]
I happened to be using the "authors" variable in the program, but `package-lint` gives me the following errors:

package-lint (using version 20200906.512):
```
2 issues found:
567:10: error: `authors' was removed in Emacs version 25.1.
727:10: error: `authors' was removed in Emacs version 25.1.
```
Check 

https://github.com/chenyanming/calibredb.el/blob/develop/calibredb-utils.el#L567
https://github.com/chenyanming/calibredb.el/blob/develop/calibredb-utils.el#L727

**Comment by purcell** [comment]
Yes, this is a false negative, but we don't completely expand and evaluate the code, so there's no general way for us to avoid it.

Whatever the case, you could consider this a hint that you're shadowing a built-in variable, and it might be safer to use a different variable name locally in that function.
